src/pages/registerUser.vue

```diff
@@ -0,0 +1,109 @@
+                  </v-text-field>
+                </v-col>
+                <v-col cols="12">
+                  <v-text-field :rules="[ requiredField , emailRules].flat()" v-model="signUser.email" label="Email" outlined></v-text-field>
+                </v-col>
+                <v-col cols="12">
+                  <v-row class="mb-2">
+                    <v-col cols="11">
+                      <v-text-field :rules="[ requiredField , passwordRules ].flat()" v-model="signUser.password"
+                        :type="hidePassword ? 'text' : 'password'" label="Password" outlined required>
+                      </v-text-field>
+                    </v-col>
+                    <v-col cols="1" class=" mt-4">
+                      <v-icon @click="togglePassword" :icon="hidePassword ? 'mdi-eye' : 'mdi-eye-off'">
+                      </v-icon>
+                    </v-col>
+                  </v-row>
+                </v-col>
+                <v-col cols="12">
+                  <v-text-field :rules="confirm" v-model="signUser.confirmPassword" label="Confirm Password"
+                    outlined type="password">
+                  </v-text-field>
+                </v-col>
+                <v-col cols="12">
+                  <v-file-input :rules="[requiredField , profileRules].flat()" label="Upload Profile Photo"
+                    @change="handleFileChange"></v-file-input>
+                </v-col>
+                <v-col cols="12">
+                  <v-alert v-if="userExist" type="error" class="mt-2">
+                    User already exists. Please choose a different email.
+                  </v-alert>
+                </v-col>
+                <v-col cols="12">
+                  <v-btn class="mt-2 pointer ml-10" color="blue" type="submit" dark :disabled="hasErrors">
+                    <div v-if="!isLoading">
+                      User Register</div>
+                    <div v-else>
+                      <v-progress-circular indeterminate></v-progress-circular>
+                    </div>
+                  </v-btn>
+                </v-col>
+              </v-row>
+              <v-col v-if="sucessModal" cols="12">
+                <sucessfullSignUp />
+              </v-col>
+            </v-form>
+          </v-card-text>
+        </v-card>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+  
+<script setup>
+import { computed, onMounted } from 'vue';
+
+import { signUpApi } from '../composables/loginSignup.js';
+import sucessfullSignUp from '../components/sucessfullUserRegisterModal.vue'
+import { confirmPasswordRules , requiredField , nameRules , phoneRules , emailRules , passwordRules , profileRules } from "../composables/validationRules"
+
+const { createAccount, signUser, togglePassword, hidePassword, sucessModal, userExist, isLoading } = signUpApi();
+
+const handleFileChange = (event) => {
+  const file = event.target.files[0];
```

Comment:
also if your not doing anything with file variable, then do not create it @ShivanshSinghnnovative